google_compute_engine_oslogin/bin/google_oslogin_control

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This might cause problems on bsd which doesn't have bash by default.

Reply:
i'll need to do some BSD testing then because some of the code here is probably bash-specific.

Reply:
i have begrudgingly restored sh(1) compatible syntax